app/controllers/sessions_controller.rb

```diff
@@ -4,9 +4,15 @@ def new; end
 
   # POST  : /login
   def create
-    user = User.find_by email: params.dig(:session, :email)&.downcase
-    if user&.authenticate params.dig(:session, :password)
+    user = find_user
+    if authenticated? user
       log_in user
+      if params.dig(:session,
+                    :remember_me) == Settings.development.params.to_s
```

Comment:
```suggestion
                    :remember_me) == Settings.development.params.remember_me
```
Chỗ này đặt tên cho tường minh e nhé